docs/api.mdx

```diff
@@ -64,11 +65,19 @@ This is useful for sending multiple transactions that depend on each other.
 
 **Refunds:**
 
-- Refunds are based on the last transaction in the bundle.
-- `refundPercent` field
+When a searcher makes money by using some user tx, it pays the builder to be included in the block. Part of that payment is refunded to the user.
+To determine how much the searcher paid the builder, we check the delta in the builder's balance before and after executing a particular tx in the bundle. By checking this delta, we are considering gas payments and also explicit transfers from the searcher to the builder.
+ 
+ These refunds should not be confused with [BuilderNet Refunds](./refunds.mdx).
```

Comment:
```suggestion
 These refunds should not be confused with [Gas Fee Refunds](./refunds.mdx).
```
Let's use this terminology, which we already use [here](https://docs.flashbots.net/flashbots-auction/advanced/gas-fee-refunds#:~:text=Searchers%20and%20private%20transaction%20API,to%20be%20eligible%20for%20them.)